lib/atomic_web/components/avatar.ex

```diff
@@ -0,0 +1,49 @@
+defmodule AtomicWeb.Components.Avatar do
+  @moduledoc false
+  use AtomicWeb, :component
+
+  attr :src, :string, default: nil
+  attr :name, :string, required: true
+  attr :type, :atom, values: [:user, :company], default: :user
```

Comment:
```suggestion
  attr :type, :atom, values: [:user, :company], default: :user
```
What about organizations?

Reply:
Organizations are companies, I chose to call it that name because its also the same look for partner avatars